src/prefect/settings.py

```diff
@@ -394,6 +394,13 @@ def warn_on_database_password_value_without_usage(values):
     description="""The maximum size in bytes for a single log.""",
 )
 
+PREFECT_LOGGING_STYLED_CONSOLE = Setting(
+    bool,
+    default=True,
+    description="""Whether to style console logs.""",
+    value_callback=debug_mode_log_level,
+)
```

Comment:
This matches the copy of the `PREFECT_CLI_COLORS` setting.
```suggestion
PREFECT_LOGGING_COLORS = Setting(
    bool,
    default=True,
    description="""If `True`, use colors in logging output. If `False`,
        output will not include colors codes. Defaults to `True`.""",
    value_callback=debug_mode_log_level,
)
```